### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by Ash-Pera**
Added `-O2` optimization flag, since program is relativity intensive (seems to still work ok with cursory testing). If debugging, flag should probably be `-Og`. Few typo fixes in various print statements. Couple of minor improvements to utils.py

### CODE REVIEW

**Comment by ftherrien** [comment]
I love it! Is there a reason you chose O2 over O3?

File: p2ptrans/utils.py [hunk 1]
```diff
@@ -40,16 +41,14 @@ def find_uvw(stretch_dir, basis = np.eye(3)):
...
-         for j in np.arange(-10,10):
-            for k in np.arange(-10,10):
-               if [i,j,k] != [0,0,0]:
-                  unit_vec = basis.dot(np.array([i,j,k]))
-                  unit_vec = unit_vec/la.norm(unit_vec)
-                  cur_dist = la.norm(unit_vec-vec)
-                  if cur_dist < min_dist[l]:
-                     min_dist[l] = cur_dist
-                     min_uvw[:,l] = np.array([i,j,k], np.int).T
+      for i, j, k in product(np.arange(-10,10), repeat=3):
```
Comment (ftherrien) [inline p2ptrans/utils.py:44]: I like it! There are probably other places in the code that could use this!

**Comment by Ash-Pera** [comment]
No particular reason... in my experience O2 is the sort of 'standard' optimization. O3 usually produces faster, if slightly larger code. There's a variety of edge cases where O3 is slower (lots of subroutines to inline, small L1 caches) and in theory it's a little buggier (if you're relying on undefined behavior ;) ) but it might be more appropriate.

**Comment by ftherrien** [comment]
Ok yes that makes sense. The code actually uses -O3 right now. When you install p2ptrans the code is compiled following the instructions in setup.py:

https://github.com/ftherrien/p2ptrans/blob/e6345cf57a876cbd126db2b27ebe06a0d55c1477/setup.py#L4

I use the makefile for developpement only so that I don't have to reinstall everytime I modify fortran code. Since it is for developement only I replaced it with -Og as you suggested.

**Pull request merged**